src/sels_fe/src/components/lesson/Lesson/CategoryList.jsx

```diff
@@ -9,22 +8,25 @@ import Lessons from "./components/Lessons"
 const CategoryList = ({ user }) => {
   const quiz_list = useSelector((state) => state.Quiz.quiz_list)
   const lesson_list = useSelector((state) => state.Lesson.lesson_list)
+  const authUser = useSelector((state) => state.AuthUser.data)
```

Comment:
I have not noticed this before.
When naming variables or functions, please always use camelCase in javascript. Currently it is inconsistent.
Pls check other variables. If its too many, then you can update gradually in your next PRs

```suggestion
  const quizList = useSelector((state) => state.Quiz.quiz_list)
  const lessonList = useSelector((state) => state.Lesson.lesson_list)
  const authUser = useSelector((state) => state.AuthUser.data)
```